assets/css/main.css

```diff
@@ -317,18 +317,14 @@ Markdown - Can be used to auto apply styling to tags within the .markdown class
     @apply mt-11;
 }
 
-.markdown h2:after {
```

Comment:
What happened to the H2 as part of this?

Reply:
I may need to rebase...